unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarNamesHUD/AvatarNamesHUDController.cs

```diff
@@ -1,11 +1,14 @@
 using System.Collections.Generic;
 using DCL;
+using UnityEngine;
 
 namespace AvatarNamesHUD
 {
     public class AvatarNamesHUDController : IHUD
     {
         private BaseDictionary<string, Player> otherPlayers => DataStore.i.player.otherPlayers;
+        private BaseVariable<bool> avatarNamesVisible => DataStore.i.HUDs.avatarNamesVisible;
```

Comment:
> That's a nice improvement! Good job! However, I had entered the build and I am unable to see the names of the avatars.
> 
> There was invisible since the beginning
> 
@Suduck 
Dammit, that's probably the default value for the opacity slider (which is 0)